EmployeeSalary.Api/V1/Controllers/SalaryCalculationController.cs

```diff
@@ -0,0 +1,70 @@
+using EmployeeSalary.Api.Models.Entities.ConcreteEmployee;
+using EmployeeSalary.Api.V1.DTOs;
+using EmployeeSalary.Api.V1.Services;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+
+namespace EmployeeSalary.Api.V1.Controllers;
+
+[ApiController]
+[Route("api/v1/employee/salary-calculation")]
+public class SalaryCalculationController : ControllerBase
```

Comment:
controllers usually are based on entities not functionality.
rename this to EmployeeController.
there's something similar to what you've done, check this link if you want to read more about it. but for now it's out of the task scope.
https://github.com/ardalis/ApiEndpoints